java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -785,6 +785,53 @@ public Map<String, String> getConfigEntries() {
       return Collections.unmodifiableMap(configEntries);
     }
 
+    /**
+     * Adds in the supplied externally loaded initializers.
+     *
+     * <p>Note the initializers are copied into the session once it has been created, and the native
+     * references are removed from this {@code SessionOptions}. Once the session has been created
+     * those initializers can be closed. This is a different lifetime to initializers added via
+     * {@link #addInitializer(String, OnnxTensorLike)}. The initializers must be created from 
+     * {@link java.nio.Buffer} objects.
+     *
+     * @param initializers The map of names to initializers.
+     * @throws OrtException If the initializers could not be loaded.
+     */
+    public void addExternalInitializers(Map<String, OnnxTensorLike> initializers)
+        throws OrtException {
+      checkClosed();
+      if (!initializers.isEmpty()) {
```

Comment:
Minor nit, but could invert the logic to make it early exit vs having the rest of the function in the if{} block?

Reply:
Like this:
```java
      checkClosed();
      if (initializers.isEmpty()) {
        return;
      }
      String[] names = new String[initializers.size()];
      long[] handles = new long[initializers.size()];
      int i = 0;
      for (Map.Entry<String, OnnxTensorLike> e : initializers.entrySet()) {
        names[i] = e.getKey();
        handles[i] = e.getValue().nativeHandle;
        i++;
      }
      addExternalInitializers(OnnxRuntime.ortApiHandle, nativeHandle, names, handles);
``` ?

Reply:
Yep, up to you, just something I've found cleaner and avoids indenting deeper as the function goes, and makes it clear there's no 'else'.

Reply:
Done.